src/spec/page-objects/template-page.js

```diff
@@ -0,0 +1,103 @@
+class TemplatePage {
+  constructor() {
+    this._templateNameLabel = $("form .templates--information label[for='uName'");
+    this._templateFromEmailLabel = $("form .templates--information label[for='uFromEmail'");
+    this._templateFromNameLabel = $("form .templates--information label[for='uFromName'");
+    this._templateSubjectLabel = $("form .templates--information label[for='uSubject'");
+    this._templateEditorDescription = $("form .templates--edition label[for='uContent']");
+    this._templatePageTitle = $('form .templates--information h2');
```

Comment:
I would remove this kind of elements, we do not need to read a title, right?

Reply:
👍